File: JavaScript/1114/senhaFixa.js
let input = require("fs").readFileSync("1114/stdin", "utf8");
let lines = input.split("\n").map(parseInt);

function senhaFixa() {
  let senha = 2002;

  while (senha !== lines[0]) {
    console.log("Senha Invalida");
    break;
  }

  if (senha === lines[0]) {
    console.log("Acesso Permitido");
  }
}

senhaFixa();

File: Java/src/pkg1114/senhaFixa.java
package pkg1114;
import java.util.Scanner;

public class senhaFixa{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int senha;
        do {
            senha = scanner.nextInt();
            if (senha != 2002){
                System.out.println("Senha Invalida");
            }
            else{
                System.out.println("Acesso Permitido");
            }

        } while(senha != 2002);

    }
}
